### DISCUSSION TITLE
[委托]Fairy ARG

### DISCUSSION THREAD

**Discussion by SakuraLainMaid** [share121/inter-knot#4778] [委托]
有人解出来了吗，在线等挺急的（

**Comment by lllliufeng** [comment]
1.aHR0cHM6Ly9wYW4uYmFpZHUuY29tL3MvMUhoblRvTm1yV2QxRFhLTEpWelRCZlE/cHdkPXp6enoKClZ3ZHV3bHFqIGl1cnAgdmZ1ZHdmaywgd2toIGlscWRvIGRxdnpodSBsdiBrbGdnaHEgbHEgaGRmayBsd2h1ZHdscnEu
base64解码得到百度盘地址和Vwduwlqj iurp vfudwfk, wkh ilqdo dqvzhu lv klgghq lq hdfk lwhudwlrq.
后者是凯撒密码，往后移三位得到
Starting from scratch, the final answer is hidden in each iteration.
从头开始，最终的答案藏在一次次的迭代之中。

2.下载压缩包，根据Fariy第一个提示的克莱因蓝的Hex颜色代码#002fa7（注意是小写）解压文件得到图片“行列相交.jpg”
![行列相交](https://github.com/user-attachments/assets/3a124b62-aaeb-4899-a820-8a7c57a12027)
根据图片名称以及Fairy的提示，可知使用了图片混淆技术，用行+列模式解密。由于密钥在0-1之间，结合第二个提示“黄金比例”可知为0.618
解密得到
![image](https://github.com/user-attachments/assets/99e1c003-20b5-4257-8afa-168698ace0fa)

3.视频中没有了提示，所以这个谜题就需要前面提到的“最终的答案”
哪里有迭代呢？当然是Fairy的动态啦（其实我关注了Fairy，但从来没给我推送过）
懒得去找了，在Fairy主页搜索“迭代”会得到三条动态
![image](https://github.com/user-attachments/assets/072ec2ae-52c1-4c18-a427-e2c29a1c26e3)
查看评论得到

>2.一个城市，一座废墟。
>3.一个账号，一段传奇。
>4.一类元素，一种能源。

我们发现没1.啊？但是早在
一个多星期前，Fairy的置顶动态出现了“1.一只邦布，一个化身。”的评论，这就是全部线索。
根据游戏设定，很容易确定答案是：“伊埃斯（Eous）”，“艾利都（Eridu）”，“法厄同（Phaeton）”，“以太（Ether）”
根据提示的字符得到[这个](http://www.ouph.top/)网址，跳转到qq群

4.最后就是伊夫·克莱因，全篇唯一提到的人的生日1962年6月6日啦<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_5477506439621383475.png" alt="阿姬开心" width="75" >

可能用到的工具：
[Base64加解密](https://www.base64decode.org/zh/)
[凯撒密码加解密](https://www.lddgo.net/encrypt/caesar-cipher)
[图片混淆加解密](https://picencrypt.sky4k.top/)
[Fairy的主页](https://space.bilibili.com/3546720545803162)
[伊夫·克莱因百度百科](https://baike.baidu.com/item/%E4%BC%8A%E5%A4%AB%C2%B7%E5%85%8B%E8%8E%B1%E5%9B%A0/1862013)

**Reply by lllliufeng** [reply]
也确实没注意动态推送，所以解3时看了提示.总体还是比较简单的，手法都较为常见<img src="https://img-static.mihoyo.com/communityweb/upload/8aee1ff2c6508b43813237163cacec1c.png" alt="三月七_加油" width="75" >
希望下次还有~

**Reply by SakuraLainMaid** [reply]
Just kidding，我是top20中的一个
